dev/InfoBar/InfoBar.idl

```diff
@@ -0,0 +1,143 @@
+﻿namespace MU_XC_NAMESPACE
+{
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+enum InfoBarCloseReason
+{
+    CloseButton = 0,
+    Programmatic = 1
+};
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+enum InfoBarSeverity
+{
+    Informational = 0,
+    Success = 1,
+    Warning = 2,
+    Critical = 3,
+};
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+unsealed runtimeclass InfoBarClosedEventArgs
+{
+    InfoBarCloseReason Reason { get; };
+}
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+unsealed runtimeclass InfoBarClosingEventArgs
+{
+    InfoBarCloseReason Reason { get; };
+    Boolean Cancel { get; set; };
```

Comment:
Does this need a GetDeferral method?

Reply:
We discussed this during the API Review and decided against it: https://github.com/microsoft/microsoft-ui-xaml-specs/pull/94#discussion_r487467273